src/Servers/Kestrel/Core/src/Internal/Infrastructure/StringUtilities.cs

```diff
@@ -109,6 +112,127 @@ public static unsafe bool TryGetAsciiString(byte* input, char* output, int count
             return isValid;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
+        public unsafe static bool BytesOrdinalEqualsStringAndAscii(string previousValue, Span<byte> newValue)
+        {
+            // We just widen the bytes to char for comparision, if either the string or the bytes are not ascii
```

Comment:
nit: another "comparision" here (and some other lines below). Might need 'find & replace all' 😉

Reply:
Went for search and replace :)